oops/oops.go

```diff
@@ -469,3 +470,61 @@ func Recover(p interface{}) error {
 	}
 	return wrapf(fmt.Errorf("recovered panic: %v", p), "")
 }
+
+// MultiError acts as a container to capture multiple errors and their tracebacks
```

Comment:
Nit: Because `multiError` is a lower cased, non-exported struct; its documentation should reflect that.

```suggestion
// multiError acts as a container to capture multiple errors and their tracebacks
```

Reply:
Ah yeah I had it as an exported value previously but reverted to a private member, will update